lib/tasks/dev.rake

```diff
@@ -0,0 +1,48 @@
+namespace :server do
+  desc "Configures and starts dependent servers"
+  task initialize: :environment do
+    Rake::Task["db:create"].invoke
+    Rake::Task["db:migrate"].invoke
+  end
+
```

Comment:
This rake task doesn't work for me. If I drop the exisiting databases from postgres I'm not able to create a dev and a test db using the rake task.

Reply:
I am encountering these errors as well. I fixed this using https://github.com/pulibrary/marc_liberation/pull/930/files#diff-11dd86092c3c21bdb579844c51b327932d6364d851f3f3323af3e202d1758d8bR10